theano/tensor/basic.py

```diff
@@ -5098,31 +5098,41 @@ def flatten(x, outdim=1):
         x : theano.tensor.var.TensorVariable
             the variable that should be reshaped.
 
-        outdim : int
+        ndim : int
             the number of dimensions of the returned variable
```

Comment:
Maybe add that the default is equivalent to 1.
It was showing in the default value of the function before, but now it shows "None" instead.